source/administration/mobile-changelog.md

```diff
@@ -1,5 +1,26 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.39.0 Release
+- Release Date: January 16, 2021
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.25 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.19 has ended and upgrading to server ESR v5.25 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Teams in the sidebar are now ordered by user preference.
+ - Typing an emoji in a post now adds the emoji to the list of recently used emojis.
+
+### Bug Fixes
```

Comment:
Were there no other bug fixes in v1.39? Seems like a really tiny release

Reply:
I added a few more that were recently fixed, but yes it's a smaller release.